crates/democracy/src/lib.rs

```diff
@@ -1069,11 +1085,14 @@ impl<T: Config> Pallet<T> {
         let r = last.saturating_sub(next);
 
         // pick out another public referendum if it's time.
-        if (now % T::LaunchPeriod::get()).is_zero() {
+        let current_time = T::UnixTime::now();
```

Comment:
To confirm, this reads from `pallet-timestamp` right?

Reply:
yes